tests/components/demo/test_geo_location.py

```diff
@@ -53,26 +48,21 @@ def test_setup_platform(self):
                 if state.domain != geo_location.DOMAIN:
                     # ignore home zone state
                     continue
-                assert (
-                    abs(state.attributes["latitude"] - self.hass.config.latitude) < 1.0
-                )
-                assert (
-                    abs(state.attributes["longitude"] - self.hass.config.longitude)
-                    < 1.0
-                )
-                assert (
-                    state.attributes["unit_of_measurement"]
-                    == DEFAULT_UNIT_OF_MEASUREMENT
-                )
+                assert (abs(state.attributes["latitude"] -
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4636028)